anonymizer_module/opmon_anonymizer/tests/manual/drop_example_table.py

```diff
@@ -1,27 +1,31 @@
-import psycopg2 as pg
+import logging
 import os
 import sys
 
+import psycopg2 as pg
+from settings import postgres  # type: ignore
```

Comment:
isn't settings a local module? I'm referring to the import order

Reply:
Yes, it is local import. Fixed.